app/views/catalog/_component_overview.html.erb

```diff
@@ -0,0 +1,10 @@
+<% document ||= @document %>
```

Comment:
I don't believe this is necessary since we're using `@document` everywhere below.